### PR TITLE
feat: add pagination to the search page

### PR DESCRIPTION
**Pull request by cindyli**
* [X] This pull request has been linted by running `npm run lint` without errors
* [X] This pull request has been tested by running `npm run start` and reviewing affected routes
* [X] This pull request has been built by running `npm run build` without errors
* [X] This isn't a duplicate of an existing pull request

## Description

Add the pagination to the search result page.

## Steps to test

1. Search by a keyword, for example, an empty space " ";
2. The search status text shows: Search results: " " (Note the double quotes around the empty character);
3. Only 10 results show up;
4. The pagination is shown underneath the result. Clicking through some of them to check if page url and content are directed correctly.

### CODE REVIEW

File: src/search.njk [hunk 1]
```diff
@@ -5,43 +5,104 @@ permalink: /search/
...
+		<symbol id="previous" width="14" height="24" viewBox="0 0 14 24">
+			{% include 'svg/prev.svg' %}
+		</symbol>
+		<symbol id="next" width="14" height="24" viewBox="0 0 14 24">
+			{% include 'svg/next.svg' %}
+		</symbol>
+	</svg>
+
+	<article class="search">
+		<h1>Search results: "{{ '{{ searchQuery }}' }}"</h1>
```
Comment (TeddTech) [inline src/search.njk:21]: This should use curly [double quotes](https://practicaltypography.com/straight-and-curly-quotes.html) `“”`

The same should be done for the _Tag_ page

**Comment by cindyli** [comment]
@TeddTech, thanks for the comment. It has been addressed and ready for another look.

File: src/_includes/components/pagination.njk [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+		</symbol>
+		<symbol id="next" width="14" height="24" viewBox="0 0 14 24">
+				{% include 'svg/next.svg' %}
+		</symbol>
+</svg>
+
 <nav class="pagination" aria-label="pagination">
 	<ul class="pagination-list">
-		{%- if pagination.href.previous %}
+		{# {{ pagination.pageNumber }} #}
```
Comment (greatislander) [inline src/_includes/components/pagination.njk:12]: Is this needed?

**Comment by TeddTech** [comment]
> @TeddTech, thanks for the comment. It has been addressed and ready for another look.

@cindyli should double-quotes be around Tag queries as well? 

Screen Shot 2020-05-07 at 8 52 43 AM

**Comment by greatislander** [comment]
> @cindyli should double-quotes be around Tag queries as well?

@TeddTech This PR only impacts the search page— please open a PR to address the tag title separately.

**Comment by cindyli** [comment]
> > @cindyli should double-quotes be around Tag queries as well?
> 
> @TeddTech This PR only impacts the search page— please open a PR to address the tag title separately.

Good catch and it was a simple fix so I've addressed it in the new commit. Thanks guys.

**Comment by cindyli** [comment]
All comments have been addressed and this pull request is ready for another look. Thanks.

**Pull request merged**